arches/app/src/components/ControlledListManager/ListCharacteristic.vue

```diff
@@ -0,0 +1,141 @@
+<script setup lang="ts">
+import { computed, inject, ref } from "vue";
+import { useGettext } from "vue3-gettext";
+
+import InputText from "primevue/inputtext";
+import { useToast } from "primevue/usetoast";
+
+import { postListToServer } from "@/components/ControlledListManager/api.ts";
+import { displayedRowKey } from "@/components/ControlledListManager/const.ts";
+
+import type { DisplayedListRefAndSetter } from "@/types/ControlledListManager";
+
+const props = defineProps<{
+    editable: boolean;
+    label: string;
+}>();
+const { displayedRow: list } = inject(displayedRowKey) as DisplayedListRefAndSetter;
+
+const editing = ref(false);
+const disabled = computed(() => {
+    return !props.editable || !editing.value;
+});
+
+const formValue = ref("");
+
+const inputValue = computed({
+    get() {
+        return list.value!.name;
+    },
+    set(newVal: string) {
+        formValue.value = newVal;
+    },
+});
+
+const toast = useToast();
+const { $gettext } = useGettext();
+
+const onSave = async () => {
+    editing.value = false;
+    const originalValue = list.value!.name;
+    list.value!.name = formValue.value;
+    const success = await postListToServer(list.value, toast, $gettext);
+    if (!success) {
+        list.value!.name = originalValue;
+    }
+};
+
+const onCancel = () => {
+    editing.value = false;
+    formValue.value = list.value!.name;
+};
+</script>
+
+<template>
+    <div class="characteristic">
+        <h4>{{ props.label }}</h4>
+        <!-- TODO https://github.com/archesproject/arches/issues/10847 -->
```

Comment:
nit: please add your handle to this comment for easy reference

Reply:
I'm not so eager to add initials to code; I don't want to encourage a sense of "code ownership". We all maintain this code together; the author's name is already in the blame--we don't also need it in the code.

Reply:
fair enough. It's a holdout from a previous pattern I was once held to